app/src/main/java/com/asfoundation/wallet/promo_code/bottom_sheet/PromoCodeBottomSheetViewModel.kt

```diff
@@ -36,53 +34,39 @@ class PromoCodeBottomSheetViewModel(
   }
 
   private fun getCurrentPromoCode() {
-    observeCurrentPromoCodeUseCase()
-        .asAsyncToState() {
-          copy(promoCodeAsync = it)
-        }
-        .scopedSubscribe() { e ->
+    observeCurrentPromoCodeUseCase().asAsyncToState() {
+      copy(promoCodeAsync = it)
```

Comment:
I don't find this code style cleaner to read.
I think `asAsyncToState` inline with the `repeatableScopedSubscribe` easier to understand quickly what's going on.

Reply:
I agree, it's just code style changes (I had an option enabled that retroactively fixed line breaks -- normally this is disabled).